### DISCUSSION TITLE
compile with debug failed

### DISCUSSION THREAD

**Discussion by shutaozhenzhen** [IENT/YUView#589] [Q&A]
https://github.com/shutaozhenzhen/YUView/actions/runs/10490134511/job/29057133771

**Comment by ChristianFeldmann** [comment]
Hi! 
So if you build a debug build then the system will not be able to package the installer. That is expected.

**Reply by shutaozhenzhen** [reply]
So how to compile a debug mode in github action? what is the difference?

**Reply by ChristianFeldmann** [reply]
You can just comment out the `Release Installer` step in the `Build.yml` file. Its all the way at the bottom.

**Reply by shutaozhenzhen** [reply]
> You can just comment out the `Release Installer` step in the `Build.yml` file. Its all the way at the bottom.

still do not understand.

the compile is not dead in this part

**Reply by ChristianFeldmann** [reply]
What do you mean with "the compile is not dead"? The compilation succeeds of course. The step that fails is the following step of "Release Installer" which packages up the installers. That does not work with a debug build. So if you comment those in the `Build.yml` file then the whole workflow will succeed.

**Reply by shutaozhenzhen** [reply]
> What do you mean with "the compile is not dead"? The compilation succeeds of course. The step that fails is the following step of "Release Installer" which packages up the installers. That does not work with a debug build. So if you comment those in the `Build.yml` file then the whole workflow will succeed.

I provided the link.
If you look at the link content, you can see that it stoped at 'Build'.
The error print is `Error: dependent 'D:\a\YUViewQt\YUViewQt\Qt\lib\Qt6Widgetsd.lib' does not exist.`

**Reply by ChristianFeldmann** [reply]
Aah I looked at the wrong place. You are right. The compilation does not work since there is no debug version of Qt found. I am not sure how this works on windows. I built Qt myself (for licensing reasons) and of course I am only building the Release version. 
I don't know how to help here. Why do you want to build the debug version on CI? What is the point? You can always build it locally. Or the builds on the other platforms seem to be working.
